src/WidgetRegistry.ts

```diff
@@ -53,6 +53,13 @@ export function isWidgetBaseConstructor(item: any): item is WidgetBaseConstructo
 	return Boolean(item && item._type === WIDGET_BASE_TYPE);
 }
 
+function getEventType(label: RegistryLabel): RegistryLabel {
```

Comment:
Updated to expose an `action` from the `WidgetRegistryEvent`, so the type will always be the symbol or the string label and use an action of `loaded` (for now)